gms/gossiper.cc

```diff
@@ -1888,6 +1928,7 @@ future<> gossiper::do_shadow_round(std::unordered_set<gms::inet_address> nodes)
         gossip_get_endpoint_states_request request{{
             gms::application_state::STATUS,
             gms::application_state::HOST_ID,
+            gms::application_state::INSTANCE_ID,
```

Comment:
In https://github.com/scylladb/scylladb/commit/d7649a86c449fd1b3387d508425d42358775bde6#diff-82f03c5f4584c189f0cc79cba87518c72627d228b5fa01fb2ae7363741b2585cR28 we added RAFT_SERVER_ID which serves the same purpose. We should store it unconditionally (not just when raft feature is enabled), and reuse in this PR. There's no point in having two kinds of host ids.

Reply:
If you look at the time, this PoC is before RAFT_SERVER_ID. If we intend to reuse this id, we'd better rename the RAFT_SERVER_ID to something more generic.

Reply:
Sure.